### DISCUSSION TITLE
Better development server understanding

### DISCUSSION THREAD

**Discussion by Brackets-Coder** [TurboWarp/extensions#1735] [Q&A] [answered]
I'd like to be able to follow the steps on ["A better development server"](https://docs.turbowarp.org/development/extensions/better-development-server). I've installed Node and cloned the repo to my local machine, so there's no reason why I can't just start the server any time I want.

However, I'm wondering how this will affect my other MAMP server for web development of my own sites and if I can disable TW's server on demand.

What kind of conflicts can I expect to see if I'm needing to use both MAMP and TW's server?

**Comment by CST1229** [comment, marked as answer]
if they use different ports you should be fine

**Reply by Brackets-Coder** [reply]
That's what I would've guessed. Still though, is there a way to close down the server if you want to?

**Reply by CST1229** [reply]
ctrl+c in the terminal usually stops programs
